assets/script.js

```diff
@@ -110,6 +110,10 @@ function getAlbumInfo() {
         //album search result shown
         $("#album-pic").attr("src", icon);
         $("#summaryHeading").text(albumName);
+        $("#album-pic").attr("title", "Artist: " + response.album.artist);
```

Comment:
```suggestion
        $("#album-pic").attr("artist", response.album.artist);
```
Needs to have a descriptive **attribute name**, what you have will cause problems when trying to retrieve the artist name later